src/index.ts

```diff
@@ -109,39 +102,3 @@ async function isEmberCliProject(): Promise<boolean> {
 
   return !!emberCliBuildFile;
 }
-
-async function checkRequirements(
```

Comment:
if none of this is needed, that's awesome!

Reply:
yep, just do it during install.